model/src/guild/partial_member.rs

```diff
@@ -7,9 +7,11 @@ pub struct PartialMember {
     pub joined_at: Option<String>,
     pub mute: bool,
     pub nick: Option<String>,
+    pub permissions: Option<Permissions>,
     #[serde(skip_serializing_if = "Option::is_none")]
     pub premium_since: Option<String>,
     pub roles: Vec<RoleId>,
+    pub user: Option<User>,
```

Comment:
Maybe we should have an "interaction partial member" to avoid overloading the same partial member?